charts/gateway-helm/values.tmpl.yaml

```diff
@@ -18,7 +18,8 @@ global:
       pullPolicy: IfNotPresent
       # List of secrets in the same namespace of the component that can be used to pull images from private repositories.
       pullSecrets: []
-
+podDisruptionBudget:
```

Comment:
maybe keep this 0 for backwards compatibility?

Reply:
alright sounds good